_docs/_latest/use/explain.md

```diff
@@ -0,0 +1,274 @@
+---
+title: Explain
+order: 000
+---
+# {{ page.title }}
+
+This guide describes how to configure Explain for Forseti Security.
+
+Explain helps you understand the Cloud Identity and Access Management
+(Cloud IAM) policies that affect your Google Cloud Platform (GCP) resources.
+It can enumerate access by resource or member, answer why a principal has access 
+to a certain resource, or offer possible strategies for how to grant a specific 
+resource.
+
+## Before you begin
+
+To run Explain, you'll need the following:
+
+* Data collected by [Inventory]({% link _docs/latest/use/inventory.md %})
+
+## Running Explain
+
+Before you start using Explain, you'll first create and select the data model you
+want to use.
+
+To review the hierarchy of commands and explore Explain functionality, use
+`–help`.
+
+### Creating a data model
+
+Get the inventory_index_id and use it to create the data model by running
+the following command:
+
+```bash
+$ forseti model create --inventory_index_id <INVENTORY_INDEX_ID> <MODEL_NAME>
+```
+
+### Listing data models
```

Comment:
Why am I listing data models here? I created one, where would others come from, or what would be use cases for having more than one?

Reply:
Once you created the data model, unless you delete it by running the forseti model delete command, it will always remain there that's why you can have a list of data models.

Reply:
Are there use cases for keeping more than one data model? This guide is telling someone how to get started with Explain, so they create a data model and run it. What's the purpose of listing data models in this section?

Reply:
You are right, I have removed listing data models from the guide. Any more changes you would like me to make?